Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/ManipulationHandler.cs

```diff
@@ -883,6 +894,21 @@ private PointerData GetFirstPointer()
             return pointerIdToPointerMap.Values.First();
         }
 
+        private bool TryGetGripRotation(IMixedRealityPointer pointer, out Quaternion rotation)
+        {
+
+            for (int i = 0; i < pointer.Controller.Interactions.Length; i++)
+            {
+                if (pointer.Controller.Interactions[i].InputType == DeviceInputType.SpatialGrip)
```

Comment:
I think preferably, we'd be listening for the spatial grip input actions and caching / updating the value in the event handler. I'm not against fully this pattern though, since it should return the same data.